doc/modules/ensemble.rst

```diff
@@ -941,6 +941,31 @@ If no missing values were encountered for a given feature during training,
 then samples with missing values are mapped to whichever child has the most
 samples.
 
+Sample Weight Support
```

Comment:
Above in the UG we say that we don't support SW, this should be updated.

Weight -> weight?